src/Dashboard/Dashboard.php

```diff
@@ -559,7 +559,9 @@ public function getPrivate()
      */
     public function isPrivate(): bool
     {
-        $this->load();
+        if (!$this->load()) {
+            return false;
+        }
         return ($this->fields['users_id'] > 0 && $this->fields['users_id'] != Session::getLoginUserID());
```

Comment:
Maybe we could reuse check made in `getPrivate()`
```suggestion
        if ((bool)$this->getPrivate() === false) {
            return false;
        }
        return $this->fields['users_id'] != Session::getLoginUserID();
```